src/utils/request/xhr.ts

```diff
@@ -30,14 +30,14 @@ const DEFAULT_RESPONSE_TYPE : XMLHttpRequestResponseType = "json";
  * Perform an HTTP request, according to the options given.
  *
  * Several errors can be rejected. Namely:
- *   - RequestErrorTypes.TIMEOUT_ERROR: the request timeouted (took too long to
- *     respond).
- *   - RequestErrorTypes.PARSE_ERROR: the browser APIs used to parse the
+ *   - NetworkErrorTypes.TIMEOUT: the request timed out (took too long to
+ *                                  download all data).
```

Comment:
Can be re-updated now

Reply:
I'm not sure what you mean, could you explain a bit more?